index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Great job so far Joy but there are few changes you have to make;
- [ ] Kindly use the correct and complete HTML boiler plate, this will give your code structure, you can read more on HTML boiler plate.